src/utils/typography.js

```diff
@@ -39,7 +47,51 @@ export const getFontColor = ({
     light: colors.white,
     accent: colors[accent]
   };
-
   const fontColor = fontColors[variant] || fontColors.dark;
-  return getFontHue({ fontColor, primary, secondary, disabled, variant });
+
+  return getFontHue({
+    fontColor,
+    primary,
+    secondary,
+    disabled,
+    variant,
+    themed: false
+  });
+};
+
+export const getThemedFontColor = ({
+  theme,
+  variant,
+  accent,
+  primary,
+  secondary,
+  disabled
+}) => {
+  if (!theme.themeName) {
+    return getFontColor({
+      variant,
+      accent,
+      primary,
+      secondary,
+      disabled
+    });
+  }
+
+  const fontColors = {
+    dark: "onyx",
+    light: "white",
+    accent
+  };
+
+  const fontColor = getThemeValue(fontColors[variant] || "onyx")(theme);
```

Comment:
I belive you need to pass an object with theme: 
```
getThemeValue(fontColors[variant] || "onyx")({ theme })
```

https://github.com/ticketmaster/aurora/blob/master/src/utils/getThemeValue.js#L8